core/src/main/java/com/klaytn/caver/tx/model/KlayRawTransaction.java

```diff
@@ -19,21 +19,34 @@
 import com.klaytn.caver.crypto.KlaySignatureData;
 import org.web3j.utils.Numeric;
 
+import java.util.HashSet;
+import java.util.Set;
+
 public class KlayRawTransaction {
     private byte[] value;
-    private KlaySignatureData signatureData;
+    private Set<KlaySignatureData> signatureData;
 
-    public KlayRawTransaction(byte[] value, KlaySignatureData signatureData) {
+    public KlayRawTransaction(byte[] value, Set<KlaySignatureData> signatureData) {
         this.value = value;
         this.signatureData = signatureData;
     }
 
+    public KlayRawTransaction(byte[] value, KlaySignatureData signatureData) {
+        this.value = value;
+        this.signatureData = new HashSet<>();
+        this.signatureData.add(signatureData);
```

Comment:
```suggestion
        this.signatureData = new HashSet<>(Arrays.asList(signatureData));
```

Reply:
Wouldn't it be inefficient to add code to create an ArrayList object internally? what do you think?

Reply:
I think this object's cost is cheap. You can refer this(https://stackoverflow.com/questions/1552783/performance-of-arrays-aslist).